### DISCUSSION TITLE
Hacktribe is on TV

### DISCUSSION THREAD

**Discussion by BKLronin** [bangcorrupt/hacktribe#327] [General]
https://youtu.be/FIw9EUi0AjQ?si=F1z7xCaOSNjG7pbB

**Comment by ptowns** [comment]
Can't believe I missed this one, as a E2 and badgear fan I'd be all over this one. I guess the strange mpc title allowed it to slip by. Thank you!

**Reply by BKLronin** [reply]
Yes, the title was really misleading. :)
